nautobot_circuit_maintenance/templates/nautobot_circuit_maintenance/circuitmaintenance.html

```diff
@@ -143,10 +143,22 @@ <h1>Maintenance ID: {{ object }}</h1>
                         {% endif %}
                     </td>
                     <td>
-                        {{ circuit.circuit.termination_a.site }}
+                    {% if circuit.circuit.termination_a.location %}
+                        {{ circuit.circuit.termination_a.location }}
+                    {% elif circuit.circuit.termination_a.provider_network %}
+                        {{ circuit.circuit.termination_a.provider_network }}
+                    {% else %}
+                        {{ circuit.circuit.termination_a.site | placeholder }}
+                    {% endif %}
```

Comment:
Would be nice from a user experience perspective if these objects were hyperlinked (`get_absolute_url`, or in Nautobot 1.4 and later, `hyperlinked_object`)

Reply:
I've updated to at least do the hyperlink using `get_absolute_url` for now.